src/api/java/com/minecolonies/api/entity/citizen/AbstractEntityCitizen.java

```diff
@@ -148,10 +148,22 @@ public BlockPos getPosition()
         return new BlockPos(getPosX(), getPosY(), getPosZ());
     }
 
+    /**
+     * Disable vanilla's item picking stuff as we're doing it ourselves
+     */
     @Override
     public boolean canPickUpLoot()
     {
-        return true;
+        return false;
+    }
+
+    /**
+     * Disable vanilla steering logic for villagers
+     */
+    @Override
+    public boolean canPassengerSteer()
+    {
+        return false;
```

Comment:
wasn't there an elytra one too, to disable too?

Reply:
elytra is private, so can't override

Reply:
AT =D